client-src/elements/chromedash-drawer.ts

```diff
@@ -231,7 +231,7 @@ export class ChromedashDrawer extends LitElement {
   }
 
   isCurrentPage(href) {
-    return this.currentPage.startsWith(href);
+    return this.currentPage === href;
```

Comment:
Something to consider in the future: I think all of the urls on the menu have 0 or 1 query parameters now so it doesn't matter. But if that number grows, we will need to parse the urls and then [sort](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/sort) the query string parameters before comparing.

This will help in case a user types in the URL directly into the browser but with the query parameters out of order.